cmd/hlb/command/run.go

```diff
@@ -303,11 +308,83 @@ func ParseModuleURI(ctx context.Context, cln *client.Client, stdin io.Reader, ur
 		}
 		defer f.Close()
 		return parser.Parse(ctx, f)
+	case "git", "ssh":
```

Comment:
Should we use `gitssh://` or something like that instead of `git://`? It could be confusing that `ssh://` implies a Git repo, and in the future we may want to support remote access via SSH.

Reply:
Also, do we want to support `http://` and `https://` protocol schemes to access Git repos? It seems those are different from `git://`

https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols

Reply:
Yeah `gitssh://` maybe makes more sense here.

Reply:
I looked around for [other conventions](https://gist.github.com/edolstra/40da6e3a4d4ee8fd019395365e0772e7#flake-references), after reading around, I came up with this, WDYT?
- `git+https`: use https
- `git+ssh`: use ssh
- `git`: auto

Reply:
SGTM. Only thing I'm wondering is if we need a way to force the classic "git" protocol. `git+git://` would definitely be weird. I'm not sure this is used much anymore, so maybe we shouldn't specifically handle it for now.